tests/nvme_test.py

```diff
@@ -371,7 +371,7 @@ def create_ns(self, nsze, ncap, flbas, dps):
         """
         create_ns_cmd = f"{self.nvme_bin} create-ns {self.ctrl} " + \
             f"--nsze={str(nsze)} --ncap={str(ncap)} --flbas={str(flbas)} " + \
-            f"--dps={str(dps)}"
+            f"--dps={str(dps)} -v"
```

Comment:
I don't know if we want the spam the output. Usually this stuff works just fine. I let @MaisenbacherD decide :)

Reply:
Understood. Thank you.

Reply:
Right so fixed to the long version option. Thank you.